### DISCUSSION TITLE
Catalyst Feature Releases 🚀

### DISCUSSION THREAD

**Discussion by katiehoesley** [bigcommerce/catalyst#586] [Announcements]
If you're looking for a comprehensive list of the Catalyst features that are currently available, you're in the right place!  As of today, this Catalyst monorepo contains the following:

- The core **Catalyst** Next.js storefront, in [apps/core](apps/core). This is what is installed when you run the command above in [Quickstart](#quickstart).
  - By default the storefront supports an end-to-end B2C focused shopper journey, inclusive of:
    - Global Header & Footers
    - Home Page
    - Product Listing Pages (Category, Brand, Search/Faceted search, Comparison page)
    - Product Detail Pages
    - Cart
    - Headless Redirected Checkout
  - This includes end-to-end support for most features connected to Customer accounts, like:
    - Price Lists
    - Customer-specific pricing
    - Customer-specific category visibility
    - Customer-specific product visibility
  - To extend into more complex B2C and B2B scenarios, you'll want to utilize more of our [GraphQL Storefront API](https://developer.bigcommerce.com/docs/storefront/graphql).
- The storefront component library, in [packages/components](packages/components), and a [Storybook](https://storybook.js.org/) instance to view the collection.
- The BigCommerce [GraphQL Storefront API](https://developer.bigcommerce.com/docs/graphql-storefront) client, in [packages/client](/packages/client).

Stay tuned for more updates! 🚀

**Comment by Amthieu** [comment]
Hello!

First of all, thanks for Catalyst, it's really great!

I'd just like to know if there's an official roadmap for future features to be released? Mainly the core missing ones such as the account tabs which are not fetching any data from BigCommerce.

Thank you
